### DISCUSSION TITLE
Bring Accessories Into Clothing

### DISCUSSION THREAD

**Discussion by TimMarvelHero** [tersiswilvin/Roblox-Legacy-Old-Theme#65] [Ideas]
I can't change accessories in the avatar section with this extension. I can't find where the accessories are located so for example if I wanted to change my hat I can't, I can't find the accessories location in any category. Clothing only has the 3D clothing. In the old theme accessories were available in clothing.

**Comment by ghost** [comment]
Please turn five column avatar editor off and try again, as @tersiswilvin explained here: https://github.com/tersiswilvin/Roblox-2019-Old-Theme/issues/50#issuecomment-1984898881

![image](https://github.com/tersiswilvin/Roblox-2019-Old-Theme/assets/77752310/aa9e063f-a774-4e53-9c5f-ea59c697341c)

**Reply by TimMarvelHero** [reply]
Thanks so much you're a lifesaver.
